odoo/addons/base/tests/__init__.py

```diff
@@ -35,7 +35,6 @@
 from . import test_translate
 #import test_uninstall  # loop
 from . import test_user_has_group
-from . import test_view_validation
```

Comment:
what happens with the `/tests/test_view_validation.py` file? why not delete it? 🤔

Reply:
@mreficent thanks for pointing that out!